doc/tutorial/iris-dnn.md

```diff
@@ -50,17 +48,75 @@ Putting it all together, we have our first SQLFlow training statement.
 
 ```sql
 %%sqlflow
-SELECT *
-FROM iris.train
-TO TRAIN DNNClassifier
-WITH
+SELECT * FROM iris.train TO TRAIN DNNClassifier WITH
   model.n_classes = 3,
   model.hidden_units = [10, 10],
-  train.epoch = 100
+  train.epoch = 10
+COLUMN sepal_length, sepal_width, petal_length, petal_width
+LABEL class
+INTO sqlflow_models.my_dnn_model;
+```
+
+Under normal conditions, the training statement will complete after several minutes.
+
+## Tune
+
+Typically, the above training statement will output something like:
+
+```python
+{'accuracy': 0.4, 'average_loss': 1.0920922, 'loss': 1.0920922, 'global_step': 1100}
+
+```
+which seems to be not ideal. 
```

Comment:
The average loss doesn't look very good; an idea value should about .....  Let us see what we can do to improve model quality.

Reply:
Done.